packages/ec-linear/examples/hiff/main.rs

```diff
@@ -77,7 +77,7 @@ fn main() -> Result<()> {
 
     let num_test_cases = 2 * bit_length - 1;
 
-    let selector = Weighted::new(Best, 1)
+    let selector = DynWeighted::new(Best, 1)
```

Comment:
Question: Do we want `DynWeighted` or just `Weighted` in the examples?

Reply:
We'll address this in #263.